webrtc/integration_test/test_videoroom/assets/js/stats.js

```diff
@@ -4,8 +4,9 @@
 const checkInterval = 200;
 
 function detectBrowser() {
-  if (typeof InstallTrigger !== undefined) return "firefox";
   if (window.chrome !== undefined) return "chrome";
+  if (typeof InstallTrigger !== undefined) return "firefox";
```

Comment:
Is this on purpose?

Reply:
Yup, based on my observation, this condition currently detects Chrome as Firefox. As we support only tests on Chrome, I think I should add a comment here and comment out this `if.`

Reply:
Ok, I run it again, and the test works correctly, so I return to the previous version.